code/style.css

```diff
@@ -1,4 +1,308 @@
 html {
```

Comment:
Regarding the requirements: you styled your page in many ways, you can be proud of your nice looking website, Sofia!